static/test/javascripts-legacy/helpers/svg-paths.js

```diff
@@ -0,0 +1,45 @@
+define(function() {
```

Comment:
is there a plan to move these too? feels like we should have one way of adding them...